tutorials/io/saving_games.rst

```diff
@@ -183,21 +191,24 @@ way to pull the data out of the file as well.
                 continue;
             }
 
-            // Check the node has a save function.
-            if (!saveNode.HasMethod("Save"))
+            // Check the node implements the expected interface
+            if (saveNode is ISaveable saveableNode)
```

Comment:
You could make the diff smaller by inverting the condition with `is not` and keeping the early continue as before.

```suggestion
            if (saveNode is not ISaveable saveableNode)
```